### ISSUE TITLE
Problem with tweening text to ""

### ISSUE THREAD

**Issue by MonsieurCiao** [motion-canvas/motion-canvas#1114]
**Describe the bug**
The text that is animated from some text to "" changes y-position during the animation. Might be because of the used font, didn't test that yet.

**To Reproduce**

```ts
export default makeScene2D(function* (view) {
view.add(
<Camera>
 <Txt
        fill={"7D261D"}
        text={"FRITZ"}
        fontFamily={"Playfair Display"}
        fontSize={350}
        fontStyle={"italic"}
        fontWeight={700}
        y={0}
        x={0}
        letterSpacing={10}
        ref={title}
        textAlign={"center"}
      />
</Camera>);
yield* title().text("", 0.7);
});
```

**Expected behavior**
The text should only tween to "" and not change it's position

**Console errors**
If applicable, add a screenshot of the errors displayed in the console of your browser (F12) or your terminal.

**Package versions:**
- core: 3.17.0
- two: 3.17.1
- ui: 3.17.0
- vitePlugin: 3.17.0

**Comment by Urchmaney** [comment]
I don't know if I understand the issue. But nodes are centered. so the tweening for "text" animates to the center. 
You can change this by setting `offsetX` to `-1`.
```
export default makeScene2D(function* (view) {
view.add(
<Camera>
 <Txt
        fill={"7D261D"}
        text={"FRITZ"}
        fontFamily={"Playfair Display"}
        fontSize={350}
        fontStyle={"italic"}
        fontWeight={700}
        y={0}
        x={0}
        letterSpacing={10}
        ref={title}
        textAlign={"center"}
       offsetX={-1}
      />
</Camera>);
yield* title().text("", 0.7);
});
```
Since you set `x` and `y` to be `0` the text will now start at the center instead of the text center to be at the center. 
You might need to set the `x` value to something like `-500` to get your earlier result.

check if it helps

**Comment by MonsieurCiao** [comment]
https://github.com/user-attachments/assets/b41569f3-4e41-4f73-b1c9-776899e49579

You see that the "FRITZ" text is changing y-Position, without me animating it. I am only tweening from "FRITZ" to "". I found out though, that this doesn't happen when I set the offsetY to -1 (when I set it to 1 it still happens), but I think I shouldn't have to always set the offsetY of texts to -1, just so that i can tween their content.
